package.json

```diff
@@ -66,11 +66,11 @@
   },
   "pnpm": {
     "overrides": {
+      "@sveltejs/kit>@sveltejs/vite-plugin-svelte": "^3.0.0",
```

Comment:
The override below (`"@sveltejs/vite-plugin-svelte": "workspace:^"`) would also override SvelteKit's v-p-s peer dep, causing peer dep mismatch warnings. This override pins to the original version to avoid the warning.